presto-docs/src/main/sphinx/release/release-0.204.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.203
+=============
+
+General
+-------
+* Fix a memory tracking bug that causes the memory to be overestimated for scan operators when reading ORC files.
+* Use distributed join if one side is naturally partitioned on join keys.
+* Improve performance of correlated subqueries when filters from outer query can be propagated to the subquery.
+* Improve performance for correlated subqueries that contain inequalities.
+* Add support for all geometry types in :func:`ST_Area`.
+* Add :func:`ST_EnvelopeAsPts` function.
+* Add :func:`to_big_endian_32` and :func:`from_big_endian_32` functions.
+* Add cast between ``VARBINARY`` type and ``IPADDRESS`` type.
+* Make :func:`lpad` and :func:`rpad` functions support varbinary in addition to varchar.
+* Fill in with ``NULL`` when processing arrays of different length in :func:`zip_with`.
+
+Event Listener Changes
+----------------------
+* Add resource estimates to query events.
+
+Web UI Changes
+--------------
+* Display resource estimates in Web UI query details page.
+
+Resource Group Changes
+----------------------
+* Fix a bug that caused unnecessary queuing in deployments where no resource group configuration was specified.
+
+Hive Changes
+------------
+* Disallow creating tables with unsupported partition types.
+* Support overwriting partitions for insert queries. This behavior is controlled by session property ``insert_existing_partitions_behavior``.
+* Fix an issue that causes optimized ORC writer to write stripes with excessive size for highly dictionary encoded columns.
+
+JMX Changes
+-----------
+* Track execution statistics of `AddExchanges` and `PredicatePushdown` optimizer rules.
```

Comment:
Double backticks

Reply:
This change seems to be an engine JMX stats change, not a JMX Connector change, so move this to General.